crates/vm/levm/src/call_frame.rs

```diff
@@ -174,10 +174,7 @@ impl Stack {
 
     #[inline(always)]
     pub fn swap(&mut self, index: usize) -> Result<(), ExceptionalHalt> {
-        let index = self
-            .offset
-            .checked_add(index)
-            .ok_or(ExceptionalHalt::StackUnderflow)?;
+        let index = self.offset + index;
```

Comment:
We should document that this may panic if the index is bigger than the stack limit.